.github/actions/deploy/deploy.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+set -e -u
+
+# this was based off of the deploy script in the mbta/actions/deploy-ecs
+# action.
+#
+# Register a new ECS Task Definition using the new Docker Image
+# Update the ECS Service with this new Task Definition
+#
+# Required Arguments
+# ECS_CLUSTER: ecs cluster for service being updated
+# ECS_SERVICE: ecs service containing task being updated
+# ECS_TASK_DEF: task definition to update revision for
+# DOCKER_TAG: tag for docker image to use in new task definition
+
+# set default region so we don't have to specify --region everywhere
+export AWS_DEFAULT_REGION="us-east-1"
+
+# get the contents of the template task definition from ECS 
+echo "Retrieving ${ECS_TASK_DEF}-template task definition..."
+taskdefinition="$(aws ecs describe-task-definition --task-definition "${ECS_TASK_DEF}-template")"
+
+# use retrieved task definition as basis for new revision, but replace image
+echo "Updating container image to ${DOCKER_TAG}."
+newcontainers="$(echo "${taskdefinition}" | \
+  jq '.taskDefinition.containerDefinitions' | \
+  jq --arg tag "${DOCKER_TAG}" 'map(.image="\($tag)")')"
+
+# check to make sure the secrets are included in the new container definition
+if (echo "${newcontainers}" | jq '.[0] | .secrets' | grep '^null$'); then
+  echo "Error: The container definition is missing its 'secrets' block. Deploy cannot proceed."
+  exit 1
+fi
```

Comment:
we'll have secrets in this project, so i removed the env var used to toggle this check on and off.